modules/nf-core/interproscan/main.nf

```diff
@@ -0,0 +1,52 @@
+process INTERPROSCAN {
+    tag "$meta.id"
+    label 'process_long'
+
+    conda "bioconda::interproscan=5.59_91.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/interproscan:5.59_91.0--hec16e2b_1' :
+        'quay.io/biocontainers/interproscan:5.59_91.0--hec16e2b_1' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val(out_ext)
+
+    output:
+    tuple val(meta), path('*.tsv'), optional: true, emit: tsv
+    tuple val(meta), path('*.xml'), optional: true, emit: xml
+    tuple val(meta), path('*.gff3'), optional: true, emit: gff3
+    tuple val(meta), path('*.json'), optional: true, emit: json
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def appl = "-appl TIGRFAM,FunFam,SFLD,PANTHER,Gene3D,Hamap,ProSiteProfiles,Coils,SMART,CDD,PRINTS,PIRSR,ProSitePatterns,AntiFam,Pfam,MobiDBLite"
```

Comment:
Is this a mandatory argument?

Reply:
-appl is not mandatory, but if skipped (which would imply all possible applications), it would not work. Strictly speaking, this is an error on the package side in the end, and can be considered a workaround. Anyway, I added appl option explicitly so it can be specifically overridden by the user (e.g., only analyzing for Coils,SMART)....